src/api/routes/meetingSlot.ts

```diff
@@ -0,0 +1,29 @@
+import { procedure, router } from "../trpc";
+import { z } from "zod";
+import db from "../database/db";
+
+export const meetingSlotRouter = router({
+  list: procedure.query(async () => {
```

Comment:
please follow code structure and also coding style of existing routes files (see also CodingStyle.md)